README.md

```diff
@@ -104,6 +104,18 @@ function customParsingFunction(header) {
 
 app.use(versionRequest.setVersionByAcceptHeader(customParsingFunction))
 ```
+#### Version formatting
+Before setting the version, it is always formatted, so the resulting version is a semver comaptible string, except the following cases:
+
+* if the version was set as an Object, it will be returned in stringified format (using JSON.stringify)
+* if the version has a longer format than the semver format, we attept to clean it up by removing the zeros (e.g. 1.0.0.1 will become 1.0.1), but if there are no more zeros to remove, we leave it as it is (e.g. 1.0.1.1.1 will become 1.1.1.1)
```

Comment:
It seems like a weird behavior to clean up zeroes. Instead, it should be a very simple rule of just accepting the first 3. i.e provided "1.0.0.2" it will result in "1.0.0".
Need to update the code below accordingly (I left a comment there)

Reply:
Agreed, truncation is a lot cleaner.